008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+      Config & config\{\} (see \Cref{tab:agentConfig})\\
+      CurrBehaviour & \textit{N}\\
+      MaxFloorGuess & \textit{N}\\
+      AverageFoodIntake & \textit{N}\\
+      ShortTermMemory & [\textit{N}]\\
+      LongTermMemory & [\textit{N}]\\
+      ProposedTreaties & [\textit{Treaty}] \\
+      TrustTeams & map\{ \textit{UUID}, \textit{N} \} \\
+      Neighbours & \{ above:\textit{UUID}, below:\textit{UUID} \} \\
+    \end{tabular}
+    \caption{Parameters held in the \textit{Agent} data structure}
+    \label{tab:agentStruct}
+\end{center}   
+\end{table}
+
+
+%%%%%%%%%
+%%%%%%%%% SOCIAL MOTIVES
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Social Motives}\label{social_motives}
+%%
+%% Theoretical Framework
+%%
+\subsection{Social Motives Spectrum}
+The initial concept upon which this agent's behaviour is based revolves around the four possible social motives \cite{socMot} that this agent can have based on different principles, practices, or characteristics. This in turn leads to a ``mixed-motive'' setting \cite{schellingConflict} in the tower. These social motives are: 
+
+\begin{itemize}
+    \item \textbf{Altruism:} The disinterested and selfless concern for the well-being of others. An altruist then acts in a way that purely benefits others, even if it means harming themselves.
+    \item \textbf{Collectivism:} The practice or principle of giving a group priority over each individual in it. A collectivist then acts in a way that benefits the group, themselves included, over purely the individual.
+    \item \textbf{Selfishness:} Being concerned excessively or exclusively with oneself. A selfish agent will act in a way to satisfy themselves, but not necessarily with the intent to harm the other agents.
+    \item \textbf{Narcissism:} An excessive interest or admiration of oneself. A narcissistic agent will act in a way that not only benefits themselves, but also hinders the collective.
+\end{itemize}
+
+For this implementation, we assert that all agents' social motives can be defined on a spectrum, with one end corresponding to pure altruism and the other end corresponding to pure narcissism. This is represented as a continuous value between 0.0 and 10.0 that shows where an agent lies in the social motive spectrum. Using a continuous rather than discrete variable for the social motive allows for greater expression: for example, two agents may both be referred to as collectivists, but one may be significantly closer to turning selfish than the other. A number closer to 0.0 corresponds to altruistic behaviours, and a number closer to 10.0 corresponds to narcissistic behaviour. The boundaries for the 4 distinct social motives are shown in Figure \ref{fig:socialMotives}. The initial “natural” state of a given agent as well as its current social motive are also shown and labelled \textit{baseBehaviour} and \textit{currentBehaviour}, respectively. To begin, we set \textit{currentBehaviour} = \textit{baseBehaviour}.
```

Comment:
```suggestion
For this implementation, we assert that all agents' social motives can be defined on a spectrum, with one end corresponding to pure altruism and the other end corresponding to pure narcissism. This is represented as a continuous value between 0.0 and 10.0 that shows where an agent lies in the social motive spectrum. Using a continuous rather than discrete variable for the social motive allows for greater expression: for example, two agents may both be referred to as collectivists, but one may be significantly closer to turning selfish than the other. A number closer to 0.0 corresponds to altruistic behaviour, and a number closer to 10.0 corresponds to narcissistic behaviour. The boundaries for the four distinct social motives are shown in Figure \ref{fig:socialMotives}. The initial “natural” state of a given agent and its current social motive are also shown and labelled \texttt{baseBehaviour} and \texttt{currentBehaviour}, respectively. To begin, we set \texttt{currentBehaviour} = \texttt{baseBehaviour}.
```